### PR TITLE
pdctl: add member command

### PR DESCRIPTION
**Pull request by nolouch**
PTAL @huachaohuang @siddontang @overvenus

### CODE REVIEW

File: pdctl/README.md [hunk 1]
```diff
@@ -61,5 +61,23 @@ show or set the balance config
 >> config set leader-schedule-interval 20s
 Success!
 ```
+
+#### Member [region | delete]
```
Comment (huachaohuang) [inline pdctl/README.md]: `[leader | delete]`

File: pdctl/command/member_command.go [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+	}
+	m.AddCommand(NewLeaderMemberCommand())
+	m.AddCommand(NewDeleteMemberCommand())
+	return m
+}
+
+// NewDeleteMemberCommand return a delete subcommand of memberCmd
+func NewDeleteMemberCommand() *cobra.Command {
+	d := &cobra.Command{
+		Use:   "delete <member_id>",
```
Comment (huachaohuang) [inline pdctl/command/member_command.go]: "delete <member_name>"

File: pdctl/command/member_command.go [hunk 3]
```diff
@@ -0,0 +1,120 @@
...
+		printResponseError(r)
+		return
+	}
+
+	io.Copy(os.Stdout, r.Body)
+}
+
+func deleteMemberCommandFunc(cmd *cobra.Command, args []string) {
+	if len(args) != 1 {
+		fmt.Println("Usage: member delete <member_id>")
```
Comment (huachaohuang) [inline pdctl/command/member_command.go]: ditto

File: pdctl/README.md [hunk 4]
```diff
@@ -61,5 +61,23 @@ show or set the balance config
...
+##### example
+```
+>> member
+{
+  "members": [......] 
+}
+>> member leader
+{
+  "addr": "http://192.168.199.229:2379",
+  "id": 9724873857558226554
```
Comment (siddontang) [inline pdctl/README.md]: output name

**Comment by overvenus** [comment]
Changing cluster members is dangerous, can we double check the inputs?

```
>> member delete pd2
Do you want to continue? [Y/n]
```

**Comment by siddontang** [comment]
> Do you want to continue? [Y/n]
It is unnecessary, user can still send HTTP directly to remove the member.

**Comment by huachaohuang** [comment]
LGTM

**Comment by huachaohuang** [comment]
PTAL @siddontang

**Comment by huachaohuang** [comment]
PTAL @overvenus

File: pdctl/command/member_command.go [hunk 5]
```diff
@@ -0,0 +1,120 @@
...
+		Use:   "leader",
+		Short: "show the leader member status",
+		Run:   getLeaderMemberCommandFunc,
+	}
+	return l
+}
+
+func showMemberCommandFunc(cmd *cobra.Command, args []string) {
+	url := getAddressFromCmd(cmd, membersPrefix)
+	r, err := http.Get(url)
```
Comment (siddontang) [inline pdctl/command/member_command.go]: can we extract a function to handle Get, I see many places like getLeaderMemberCommandFunc use the same code.

Comment (nolouch) [inline pdctl/command/member_command.go]: we can, I will do this in next PR.

**Comment by siddontang** [comment]
LGTM

**Pull request merged**